simulator/opendc-experiments/opendc-experiments-allocateam/tools/plot/plot.py

```diff
@@ -29,52 +29,44 @@ def reformat_large_tick_values(tick_val, pos):
 
     # make new_tick_format into a string value
     new_tick_format = str(new_tick_format)
-    
+
     # code below will keep 4.5M as is but change values such as 4.0M to 4M since that zero after the decimal isn't needed
     index_of_decimal = new_tick_format.find(".")
-    
+
     if index_of_decimal != -1:
         value_after_decimal = new_tick_format[index_of_decimal+1]
         if value_after_decimal == "0":
             # remove the 0 after the decimal point since it's not needed
             new_tick_format = new_tick_format[0:index_of_decimal] + new_tick_format[index_of_decimal+2:]
-            
+
     return new_tick_format
 
 class Plotter():
     OUTPUT_PATH = f"{os.path.dirname(__file__)}/plots/{datetime.now()}"
 
     def __init__(self, path: str):
-        self.data = pd.read_csv(path)
+        self.raw_data = pd.read_parquet(path)
```

Comment:
I think either `pyarrow` and `fastparquet` is missing from the `requirements.txt` file. Could you please include the library that works for you?